app.py

```diff
@@ -40,6 +40,7 @@
     Route("/project/{project_id:str}/import_file", PrepareFileImport),
     Route("/project/{project_id:str}/import_json", JSONImport),
     Route("/project/{project_id:str}/import/task/{task_id:str}", UploadTask),
+    Route("/project", ProjectCreationFromWorkflow),  # create fro workflow
```

Comment:
```suggestion
    Route("/project", ProjectCreationFromWorkflow),  # create from workflow
```

Reply:
But since the class is already expressive named, i would remove the comment i think

Reply:
makes sense, removed it :)